package.json

```diff
@@ -67,9 +67,9 @@
     "metro": "0.56.4",
     "metro-react-native-babel-preset": "^0.59.0",
     "react": "16.11.0",
-    "react-native": "0.62.2",
+    "react-native": "^0.65.0",
```

Comment:
this feels wrong to require 0.65+ for everything?

Reply:
This was the change that was already merged in and we asked to revert it back to 0.62 until we could fix the example app for 0.65. So it's just restoring the upgrade that was already done and approved!